tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -10560,6 +10571,279 @@ def writeBiasToGlobal(self, biasDataType, kernel, tP, gwvw, offsetVgpr, tmpSgprR
     module.add(SMovB64(dst=EXEC(), src=-1, comment="Reset exec mask"))
     return module
 
+  ########################################
+  # Amax related
+  ########################################
+  def amax_define_load_res(self) -> Module:
+    module = Module("AmaxD Set and Load")
+    module.addComment0("AmaxD Set and Load")
+
+    self.amaxVgprIdxVec = self.defineMultiVgprIndex(self.amaxVgprNames, self.amaxVgprSizes, align=1)
+    for i in range(0, len(self.amaxVgprNames)):
+      name = self.amaxVgprNames[i]
+      idx = self.amaxVgprIdxVec[i]
+      module.add(RegSet("v", "vgpr"+name, idx))
+
```

Comment:
The Amax related part is from AmaxGenerator.py part, but need some modifications.